### ISSUE TITLE
How to use --blame option?

### ISSUE THREAD

**Issue by Semigradsky** [kucherenko/jscpd#238]
How I run the command: `jscpd ./ --blame --ignore node_modules`
My output:
```
(node:13673) Warning: N-API is an experimental feature and could change at any time.
Clone found (markdown):
 - README.md [4:1 - 104:109]
   2019.md [4:1 - 104:109]

┌────────────┬────────────────┬─────────────┬──────────────┬──────────────────┬────────┐
│ Format     │ Files analyzed │ Total lines │ Clones found │ Duplicated lines │ %      │
├────────────┼────────────────┼─────────────┼──────────────┼──────────────────┼────────┤
│ markdown   │ 9              │ 755         │ 1            │ 100              │ 13.25% │
├────────────┼────────────────┼─────────────┼──────────────┼──────────────────┼────────┤
│ javascript │ 6              │ 247         │ 0            │ 0                │ 0%     │
├────────────┼────────────────┼─────────────┼──────────────┼──────────────────┼────────┤
│ json       │ 2              │ 429         │ 0            │ 0                │ 0%     │
├────────────┼────────────────┼─────────────┼──────────────┼──────────────────┼────────┤
│ yaml       │ 269            │ 10148       │ 0            │ 0                │ 0%     │
├────────────┼────────────────┼─────────────┼──────────────┼──────────────────┼────────┤
│ Total:     │ 286            │ 11579       │ 1            │ 100              │ 0.86%  │
└────────────┴────────────────┴─────────────┴──────────────┴──────────────────┴────────┘
Execution Time: 3480.919ms
```
I ran the command on this repo: https://github.com/Semigradsky/events
OS: Ubuntu 17.10
Git is installed and available in command line.

**Comment by kucherenko** [comment]
Thank you for the report, will fix it

**Comment by kucherenko** [comment]
fixed at jscpd@3.0.1

for use blame option you should use right reporter, for example:

```
jscpd tests/fixtures --blame -r consoleFull
```

or 
```
jscpd tests/fixtures --blame -r html
```
or both

```
jscpd tests/fixtures --blame -r consoleFull,html
```
